### PR TITLE
Update content for Oral History requests

### PR DESCRIPTION
**Pull request by jrochkind**
Per #2100 and #2101

Redesign based on mock-ups from @apinkney0696  linked to there. 

Requires review by @apinkney0696  and possibly other stakeholders before we merge/deploy. 

## Note on "intended use" field

Previously we had intended use as a required field on all requests, as OH staff wanted this for reporting/analysis. 

However, I agree the fewer required fields the better, only requiring it for "manual review" requests seems like an improvement. 

HOWEVER, if we are requiring this for manual review only, it makes me wonder about the prompt:

> So that we can better understand our audience, please tell us how you plan to use the oral history you have requested.

Does that accurately describe why we are requiring this field? In fact, for "manual review" items, is the answer being used to determine access/compliance with interviewee restrictions? Should the prompt say that, in some way?

Should we have a different prompt here depending on whether the item is "automatic" or "manual review"

### CODE REVIEW

**Comment by jrochkind** [comment]
Screen shots:

## Automatic access

Screen Shot 2023-06-21 at 4 06 00 PM

------------------

Screen Shot 2023-06-21 at 4 06 38 PM

## Manual review
Screen Shot 2023-06-21 at 4 07 50 PM

-------------------

Screen Shot 2023-06-21 at 4 08 23 PM

**Review by eddierubeiz — APPROVED** [review]

**Comment by apinkney0696** [comment]
Hi! I want to double-check, but I think you're right that they use this field to determine access. I will need to run any new language by Sarah and Rachel, but it's a good idea to have a different sentence under the "Intended Use" field for the manual review form. Maybe something like:

"Please tell us how you intend to use this oral history interview so that we may determine if your use complies with interviewee restrictions."

I will add comments to the original ticket and flag OH staff.

**Comment by jrochkind** [comment]
Or perhaps it can be worded less scarily, something like: ""Please tell us how you intend to use this oral history interview so that we may inform you about any relevant interviewee restrictions."

**Comment by jrochkind** [comment]
This is just waiting for sign-off from stakeholders. cc @apinkney0696

**Comment by apinkney0696** [comment]
See last comment on ticket #2100 . Going with "Please tell us how you intend to use this oral history interview to help us inform you about any relevant interviewee restrictions."

**Comment by jrochkind** [comment]
> Cleared to go with "Please tell us how you intend to use this oral history interview to help us inform you about any relevant interviewee restrictions."

Done as above.

**Pull request merged**
